.github/workflows/build_and_push_nonproduction_images.yml

```
@@ -123,4 +124,4 @@ jobs:
           echo "  docker pull ${{ steps.tags.outputs.backend_image_name }}:latest"
           echo ""
           echo -e "\033[1;36m▶️ Run Backend:\033[0m"
-          echo "  docker run --rm --env-file .env -p 8000:8000 ${{ steps.tags.outputs.backend_image_name }}:latest"
+          echo "  docker run --rm --env-file .env -p 4000:4000 ${{ steps.tags.outputs.backend_image_name }}:latest"
```

```suggestion
          echo "  docker run --rm --env-file .env -p ${{ secrets.BACKEND_PORT }}:${{ secrets.BACKEND_PORT }} ${{ steps.tags.outputs.backend_image_name }}:latest"
```

This will make sure this remains flexible and up to date.

---

cool, good idea.